pkg/e2e/verify/routes.go

```diff
@@ -36,51 +30,45 @@ func init() {
 var _ = ginkgo.Describe(routesTestName, ginkgo.Ordered, label.E2E, func() {
```

Comment:
@jbpratt Could we add the `labels.HyperShift`, this way these tests get included when the hypershift config gets called.

Reply:
added!

Reply:
Thank you @jbpratt